modules/gitops-release-notes-1-14-0.adoc

```diff
@@ -0,0 +1,106 @@
++
+For more information, refer to the link:https://argocd-operator.readthedocs.io/en/latest/usage/ha/repo-server/#high-availability-for-repo-server[HA for Repo Server].
+
+* With this update, users can implement an alternative sharding algorithm that utilizes consistent hashing with bounded loads. The algorithm ensures a uniform distribution of managed clusters across shards while minimizing the need for reshuffling during configuration changes, such as adjustments in the number of managed clusters or shards. link:https://issues.redhat.com/browse/GITOPS-3584[GITOPS-3584]
+
+* With this update, users can expose Argo CD Application labels on Kubernetes events generated by Argo CD. When events are created for applications that have specific label keys defined in the `resource.includeEventLabelKeys` field within the `argocd-cm` ConfigMap, the controller associates the relevant labels with these events. This enhancement facilitates the filtering and processing of events according to application labels. To set the `resource.includeEventLabelKeys` field in a {gitops-shortname} environment, use the `.spec.extraConfig` field in the Argo CD CR. The {gitops-shortname} Operator automatically incorporates these fields from the `.spec.extraConfig` field into the `argocd-cm` ConfigMap. link:https://issues.redhat.com/browse/GITOPS-3233[GITOPS-3233]
++
+.Example 
+[source,YAML]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: argocd-sample # <1>
+  namespace: default # <2>
+spec:
+  extraConfig:
+    resource.includeEventLabelKeys: team,env* # <3>
+----
+<1> Specifies the name of the Argo CD instance.
+<2> Specifies the namespace where the Argo CD is installed.
+<3> Specifies the label keys that match and are added onto Application events.
++
+For more information, refer to link:https://argo-cd.readthedocs.io/en/latest/operator-manual/declarative-setup/#labels-on-application-events[Labels on Application Events].
+
+* With this update, users can develop applications by verifying commit signatures for `ApplicationSet` CRD created using Git generators. Before this update, users could develop applications without the necessity of verifying commit signatures. link:https://issues.redhat.com/browse/GITOPS-3049[GITOPS-3049]
++
+[NOTE]
+====
+The signature verification does not function with the templated project field in `ApplicationSet` CRD Git Generator.
+====
+
+[id="fixed-issues-1-14-0_{context}"]
+== Fixed issues
+
+* Before this update, users could not execute a rollback for multi-source applications in the Argo CD Web UI or CLI. This update fixes the issues, and users can now carry out rollbacks for multi-source applications using the Argo CD Web UI or CLI. link:https://issues.redhat.com/browse/GITOPS-3996[GITOPS-3996]
```

Comment:
Assume they don't need to be capitalized

```suggestion
* Before this update, users could not execute a rollback for multi-source applications in the Argo CD web UI or CLI. This update fixes the issues, and users can now carry out rollbacks for multi-source applications using the Argo CD web UI or CLI. link:https://issues.redhat.com/browse/GITOPS-3996[GITOPS-3996]
```

Reply:
Fixed.